### ISSUE TITLE
navigate using CustomSend 

### ISSUE THREAD

**Issue by bkbartk** [joBr99/nspanel-lovelace-ui#224]
Is there a way to navigato to a specific page using custom send?
The main goal is to go to the detail page of the dimmer when I press one of the buttons.
So I can manage the brightness imediately
I can use tasmota rules to configure is, but I can't figure how to navigate to the detailpage.

**Comment by joBr99** [comment]
you can send `pageType~popupLight~Kueche~light.kueche_lights` to CustomSend for the detail pages

I'm doing that for the light in my kitchen, after pressing the normal light switch (connected to a shelly) it is automatically opening the detail page for the light.

**Comment by joBr99** [comment]
yeah, it cannot be used in the apps.yaml, the internal messages are documented here

https://github.com/joBr99/nspanel-lovelace-ui/tree/main/HMI
